sql-statements/sql-statement-alter-resource-group.md

```diff
@@ -0,0 +1,94 @@
+---
+title: ALTER RESOURCE GROUP
+summary: TiDB 数据库中 ALTER RESOURCE GROUP 的使用概况
+aliases: ['/docs-cn/dev/sql-statements/sql-statement-alter-resource-group/','/docs-cn/dev/reference/sql/statements/alter-resource-group/']
+---
+
+# ALTER RESOURCE GROUP
+
+`ALTER RESOURCE GROUP` 语句用于在当前所选数据库中修改资源组。
+
+## 语法图
+
+```ebnf+diagram
+AlterResourceGroupStmt:
+   "ALTER" "RESOURCE" "GROUP" IfNotExists ResourceGroupName ResourceGroupOptionList BurstableOption
+
+IfNotExists ::=
+    ('IF' 'NOT' 'EXISTS')?
+
+ResourceGroupName:
+   Identifier
+
+ResourceGroupOptionList:
+    DirectResourceGroupOption
+|   ResourceGroupOptionList DirectResourceGroupOption
+|   ResourceGroupOptionList ',' DirectResourceGroupOption
+
+DirectResourceGroupOption:
+    "RRU_PER_SEC" EqOpt stringLit
+|   "WRU_PER_SEC" EqOpt stringLit
+
+BurstableOption ::=
+    ("BURSTABLE")?
+
+```
+
+TiDB 支持以下 `DirectResourceGroupOption`, 其中 [`RU` (Resource Unit)](/tidb-RU.md) 是 TiDB 对 CPU、IO 等系统资源统一抽象的单位。
+
+| 参数           |含义                                 | 举例                    |
+|---------------|-------------------------------------|------------------------|
+
+|`RRU_PER_SEC`|每秒钟读 RU 的配额                       |`RRU_PER_SEC` = 500     |
```

Comment:
What is the definition of 1 RU?

Reply:
> What is the definition of 1 RU?

It's described in `TiDB-RU.md`. It will be finalized after coming testing.